src/styles/Home.css

```diff
@@ -147,3 +147,50 @@
 		max-height: 8rem;
 	}
 }
+
+/* ========== CurrEvents.jsx ========== */
+.section-title {
+	font-weight: bold;
+	margin: 2rem 0;
+	font-size: 2.5rem;
+	margin-bottom: 0.2rem;
+}
+
+.events-container {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 1rem;
+	margin-bottom: 2rem;
+}
+
+.event-column {
+	flex: 1 1 45%;
+	padding: 1rem;
+	border-radius: 8px;
+}
+
+.event-title {
+	text-align: center;
+	font-size: 40px;
+	font-weight: 600;
+	background-image: linear-gradient(
+		to right,
+		#c960ff,
+		#ea43d4,
+		#f499da,
+		#e16db9,
+		#c960ff
+	);
+	color: transparent;
+	background-clip: text;
+	-webkit-background-clip: text;
+	background-size: 200% auto;
+	-webkit-text-fill-color: transparent;
+	background-position: 0% center;
+	transition: background-position 2s ease;
+}
+
+.event-title:hover {
+	background-position: 100% center;
+	transition: background-position 2s ease;
+}
```

Comment:
At the bottom of this file, we should have a section for media queries to make sure that the images are centered when the screen enters a smaller window size. I've recorded a Loom here to explain better how you can test this locally, if you need some more context:

https://www.loom.com/share/5537c3fd45b74c4095783ac37fd06f52